career-mentorship-program/2021-spring/matching_code.py

```diff
@@ -506,15 +657,17 @@ def save_optimal_matches_to_csv(edges, filename):
     """
 
     with open(filename, 'w', newline='') as csvfile:
-        writer = csv.writer(csvfile, delimiter=',')   
+        writer = csv.writer(csvfile, delimiter=',')
 
         if DEBUG:
 
             """ !!! When adding new matching criteria, add to the debug output !!! """
 
             writer.writerow([
                 "Mentee email",
+                "Mentee name",
```

Comment:
We can skip adding the mentor and mentee names to the debug output. In fact we can delete the emails too. The only thing that matters is seeing the statistics on how good the match is.